spec/visual/templates/layout.erb

```diff
@@ -38,18 +38,164 @@
     #page-footer {
       border-top: 1px dotted
     }
+
+    #theme-selector {
+      display: inline-block;
+      position: relative;
+      margin: 0 auto;
+      width: 170px;
+      font-family: Arial;
+      font-size: smaller;
+      text-align: left;
+    }
+    #theme-selector select, .select-hide {
```

Comment:
I'd suggest getting rid of all of the CSS used to make the custom selector work. As explained below, this isn't necessary for this kind of site.